qadence/noise/readout.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import annotations
+
+from collections import Counter
+from enum import Enum
+from itertools import chain
+
+import numpy as np
```

Comment:
why are we working with `np.array`s in here? does it make sense to use `torch.Tensor` here for to stick to just one array/tensor type in qadence?